<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IN-PG02NJQY" failures="1" tests="1" name="runner.TestRunner" time="55.005" errors="0" timestamp="2022-03-21T22:54:54 IST" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="55.005">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at stepsdefinitions.GraphicsTest.open_arc_screen(GraphicsTest.java:63)
at âœ½.open arc screen(file:///C:/Users/rajibmahato/git/AppiumTest/AppiumExitTest/./src/test/java/features/Graphics.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
